examples/project-level-audit-log-multi/versions.tf

```
@@ -0,0 +1,11 @@
+# required for Terraform 13
+terraform {
+  required_providers {
+    google = "4.36.0"
```

We recommend pessimistic version constraint
```suggestion
    google = "~> 4.36"
```